pkg/vtpm/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# syntax=docker/dockerfile:1-labs
```

Comment:
We usually use `# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs`. Also, I don't think the comment below is really needed, we have many of our Dockerfiles using that without explanation.

Reply:
done.